CONTRIBUTING.md

```diff
@@ -0,0 +1,36 @@
+# Contributing
+
+We are always welcome your contribution!
+
+## How to contribute
+
+### Find the tasks
+
+We use GitHub issues to manage the tasks. Please find the issues you'd like to contribute to it. `welcome contribute` and `easy` are good for the first contribution.
+
+Of course, it would be great to send PullRequest which has no issue! However, please explain the motivation, the cause, the solution and so on in the description of the PR then.
+
+### Start contributing
+
+If you find the tasks you want to contribute, please comment on the issue to prevent to conflict contribution. We'll reply as soon as possible, but it's unnecessary to wait for our reaction if you have raised voice. It's okay to start contribution and send PullRequest!
+
+We've designated these issues as good candidates for easy contribution. You can always fork the repository and send a pull request (on a branch other than master).
```

Comment:
🤔 which issues? (should it be explaining about `welcome contribute` and `easy`?)

Reply:
I've just copied from the original README.md so I'm not sure which issue it is 😇

Reply:
🗡 lets delete it!